lib/telegraph/connection.rb

```diff
@@ -0,0 +1,36 @@
+module Telegraph
+  module Connection
+    BASE_URL = 'https://api.telegra.ph'
+
+    private
+
+    def connection
+      @connection ||= Faraday.new(url: BASE_URL) do |faraday|
+        faraday.request :url_encoded
+        #faraday.response :logger
+        faraday.adapter  Faraday.default_adapter
+      end
+    end
+
+    def request(method, params)
+      url = "#{BASE_URL}/#{method}"
+      params['access_token'] = token if token
+      connection.post do |req|
+        req.url(url)
+        req.headers['Content-Type'] = 'application/json'
+        req.body = params.to_json
+      end
+    end
+
+    def get(method, params)
```

Comment:
Ugh. So, `Connection#get` performs `Faraday#post`?.. MIND=BLOWN. Don't do this :)